### PR TITLE
mcobbett dependencies via maven repo

### PR DESCRIPTION
**Pull request by techcobweb**
- set-version doesnt add newlines
- correct bug in script which corrupted release.yaml
- managers component supplies release.yaml information via maven repository
- downloads the release.yaml files for manager and framework via maven dependency

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

File: dependency-download/build.gradle [hunk 1]
```diff
@@ -20,8 +20,12 @@ repositories {
 }
 
 dependencies {
+<<<<<<< HEAD
     runtime group: 'dev.galasa', name: 'dev.galasa.managers.manifest', version: version
     runtime group: 'dev.galasa', name: 'dev.galasa.framework.manifest', version: version
+=======
+    runtime group: 'dev.galasa', name: 'dev.galasa.managers', version: version
+>>>>>>> main
 }
```
Comment (AaronJhaj) [inline dependency-download/build.gradle:29]: is this correct? looks like merge conflict text?

**Review by AaronJhaj — APPROVED** [review]
approved with a comment to resolve

File: dependency-download/build.gradle (hunk 1 above)
Comment (techcobweb) [reply dependency-download/build.gradle:25]: good catch. Resolved.

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**